scripts/v6/stored-procs/02-create-scoa-batch.sql

```diff
@@ -40,7 +40,10 @@ BEGIN
 	-- The workflowType indicates what type of a financial system we're integrating with, and thus the Form_Level (i.e. form status)
 	-- at which we must create SCOA rollups. We use this determined form level value to select the appropriate SCOAJournal entries to process
 	DECLARE @formLevelValue INT;
-	IF @workflowType = 'NONE' SET @formLevelValue = 3 ELSE SET @formLevelValue = 4;
+	IF @workflowType = 'NONE' SET @formLevelValue = 3
+	ELSE IF @depreciation = 1 SET @formLevelValue = 4
+	ELSE IF @workflowType = 'PULL' SET @formLevelValue = 3
+	ELSE SET @formLevelValue = 4;
```

Comment:
I'm having second thoughts about allowing a default here. The risk may be great if the workflowType isn't specified - especially for new clients who we may have overlooked, but really don't want integration. Can we rather do a hard fail here?